app/environments/brassbirmingham/brassbirmingham/envs/classes/buildings/enums.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+class BuildingName(Enum):
+    goods = "goods"
+    cotton = ("cotton",)
+    pottery = "pottery"
+    oil = "oil"
```

Comment:
Idk where these oil bits keep coming from. Oil buildings aren't in the game, I don't think

Reply:
Yes fixed